### PR TITLE
Added file structure for pages, reducers, and store, and folder for Home

### PR DESCRIPTION
**Pull request by nicocadq**
Add file structure for react-router-dom and redux. And add some files for Home page, that would show current boards and allow navigate through them.

### CODE REVIEW

File: src/commponents/Board/Boards.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+
```
Comment (Rminetti) [inline src/commponents/Board/Boards.js]: Remove extra empty line

File: src/commponents/Board/Boards.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+const Board = () => {
+    return (
+        <h1>Hello</h1>
+    );
+}
```
Comment (Rminetti) [inline src/commponents/Board/Boards.js]: Be careful with the indentation! You should be consistent in using either 2 or 4 spaces across the project.

File: src/commponents/Home/BoardLink/BoardLink.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import styles from "../Home.module.scss";
+
+const BoardLink = (props) => {
+  const dispatch = useDispatch();
+  const { id, tittle } = props.board;
+
+  const handleDelteBoard = () => {
```
Comment (emmaielle) [inline src/commponents/Home/BoardLink/BoardLink.js:15]:
```suggestion
  const handleDeleteBoard = () => {
```

File: src/commponents/Home/BoardLink/BoardLink.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    dispatch(deleteBoard(props.board));
+  };
+
+  return (
+    <div className={styles["board-link"]}>
+      <Link to={`/board/${id}`}> {tittle} </Link>
```
Comment (emmaielle) [inline src/commponents/Home/BoardLink/BoardLink.js:21]:
```suggestion
      <Link to={`/board/${id}`}> {title} </Link>
```

File: src/commponents/Home/Home.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+const Home = () => {
+  const boards = useSelector((state) => state.app) || null;
+
+  return (
+    <div className={styles.container}>
+      <h2 className={styles.tittle}>Boards</h2>
+      <div className={styles["board-links-wrapper"]}>
+        {Object.entries(boards).map((board) => (
+          <BoardLink key={board[0]} board={board[1]} />
```
Comment (emmaielle) [inline src/commponents/Home/Home.js:18]: If in the future you might have more data you might want to pass down to BoardLink, I would send the whole board object to the `board` prop

File: src/commponents/Home/Home.module.scss [hunk 6]
```diff
@@ -0,0 +1,75 @@
+@import "../../sass/variables.scss";
+
+.container {
+  display: grid;
+  grid-template-rows: 80px 1fr;
+  height: 100%;
+  width: 100%;
+
+  .tittle {
```
Comment (emmaielle) [inline src/commponents/Home/Home.module.scss:9]: typo

**Review by emmaielle — APPROVED** [review]

**Comment by emmaielle** [comment]
maybe you should add PropTypes to handle types in the components. @Rminetti what do you think?
Also, remember to change the components folder name typo

**Comment by nicocadq** [comment]
Oh I forgot add proptypes, I'll add that.

**Pull request closed without merging**